packaging/aznfs/DEBIAN/postinst

```diff
@@ -105,9 +112,14 @@ fi
 if [ ! -f "$FLAG_FILE" ]; then
         user_consent_for_auto_update
 
+        # Start the watchdog service for NFSv3
         systemctl daemon-reload
         systemctl enable aznfswatchdog
         systemctl start aznfswatchdog
+
+        # Start the watchdog service for NFSv4
+        systemctl enable aznfswatchdogv4
+        systemctl start aznfswatchdogv4
 else
         # Clean up the update in progress flag file.
```

Comment:
note: need to make sure auto-update doesn't cause confusion as it's done from v3 watchdog process, though it's a common need for both v3 and v4.